test/DotNetEnv.Tests/EnvTests.cs

```diff
@@ -79,5 +82,28 @@ public void LoadNoClobberTest()
             DotNetEnv.Env.Load(false, false, false, true);
             Assert.Equal(Environment.GetEnvironmentVariable("URL"), "https://github.com/tonerdo");
         }
+
+        [Fact]
+        public void LoadOsCasingTest()
+        {
+            Environment.SetEnvironmentVariable("CASING", "neither");
+            DotNetEnv.Env.Load("./.env3", clobberExistingVars: false);
+            Assert.Equal(Environment.GetEnvironmentVariable("casing"), IsWindows ? "neither" : "lower");
+            Assert.Equal(Environment.GetEnvironmentVariable("CASING"), "neither");
```

Comment:
These 2 lines are the ones that actually verify the change.  All the rest just drives home the casing differences on windows